src/lib/input/autosize.ts

```diff
@@ -19,10 +19,24 @@ export class MdTextareaAutosize implements OnInit {
   /** Maximum number of rows for this textarea. */
   @Input() maxRows: number;
 
+  /** The value of the textarea. */
+  private _value: string;
```

Comment:
Is this our team's official style for these getter/setters? If it's still up for debate, I prefer:
```
/** ... */
@Input()
get value() {...}
set value(v) {...}
private _value;
```
It's a good reminder that I should be accessing it through the getter unless I have a good reason not to.